js/common/bridge_test.go

```diff
@@ -348,7 +348,7 @@ func TestBind(t *testing.T) {
 		}},
 		{"Error", bridgeTestErrorType{}, func(t *testing.T, obj interface{}, rt *goja.Runtime) {
 			_, err := RunString(rt, `obj.error()`)
-			assert.Contains(t, err.Error(), "GoError: error")
+			assert.Contains(t, err.Error(), "error")
```

Comment:
I think this can be moved to another PR that we can merge today and make this ... more focused ;)

Reply:
Sure, will do.

Reply:
See #1775.